File: Application/Services/Operations/Main/Partners/Dtos/PartnerPaymentBankAccountDto.cs

using Application.Services.Operations.Main.Partners.Enums;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Main.Partners.Dtos
{
    public class PartnerPaymentBankAccountDto:RootBaseDto
    {
        public string Institution { get; set; }
        public string Holder { get; set; }
        public string Account { get; set; }
        public string Agency { get; set; }
        public PaymentDataTypeAccountEnumDto Type { get; set; }
        public int PaymentDataId { get; set; }
        public PaymentDataDto PaymentData { get; set; }
    }
}

File: Views.old/src/components/main/partner/commons-components/dtos/partner-payment-bank-account-dto.ts
import { PaymentDataTypeAccountEnumDto } from "./enums/partner-payment-type-account-enum-dto";

export class PartnerPaymentBankAccountDto {
  id: number;
  holder: string;
  institution: string;
  account: string;
  agency: string;
  type: PaymentDataTypeAccountEnumDto;
  deleted: boolean;
  paymentDataId: number;

}
